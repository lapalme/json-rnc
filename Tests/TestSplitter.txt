[{"a":34,"c":True},{}]       {"{abc":47}
['ab',
123,"]", "ab
c",    {'a':23,'b':True}]
"ab\"c" "'apos'" 'iei"ab' 'xyz\''
