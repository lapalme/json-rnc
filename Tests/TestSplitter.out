[{"a":34,"c":True},{}]
{"{abc":47}
['ab',123,"]","ab\nc",{'a':23,'b':True}]
"ab\"c"
"'apos'"
'iei"ab'
'xyz\''
